girder_nwb/__init__.py

```diff
@@ -1,10 +1,26 @@
 from girder import plugin
+from girder import events
+
+import pluggy
+
+from .plugin_registry import get_plugin_manager
+
+hookimpl = pluggy.HookimplMarker("girder_nwb")
+
+def nwb_handler(event):
+    extensions = [i.lower() for i in event.info["file"]["exts"]]
+    if "nwb" in extensions:
+        plugin_manager = get_plugin_manager()
+        plugin_manager.hook.get_metadata(path="foobar")
+        plugin_manager.hook.get_thumbnail(path="foobar")
+    return event
 
 
 class GirderPlugin(plugin.GirderPlugin):
-    DISPLAY_NAME = 'nwb',
-    CLIENT_SOURCE_PATH = 'web_client'
+    DISPLAY_NAME = ("nwb",)
```

Comment:
This shouldn't be a tuple.  It is a bug in the cookiecutter.  Also, a more descriptive name might be better.